src/LangStrings.inc.h

```diff
@@ -455,3 +455,5 @@ DECLARE_STRING(SHIPS_DISPLAY_MODE_TOGGLE)
 DECLARE_STRING(SWITCHED_TO_ROTATIONAL)
 DECLARE_STRING(SWITCHED_TO_PARENT)
 DECLARE_STRING(L4L5_DISPLAY_MODE_TOGGLE)
+DECLARE_STRING(INCREASE_TIME_ACCELERATION)
+DECLARE_STRING(DECREASE_TIME_ACCELERATION)
```

Comment:
You also need to add the string to data/lang/core/en.json (and _no other_ .json-file after that).